packages/beacon-node/src/network/peers/discover.ts

```diff
@@ -80,12 +84,17 @@ export type SubnetDiscvQueryMs = {
   maxPeersToDiscover: number;
 };
 
+/**
+ * A map of column subnet id to maxPeersToDiscover
+ */
+type ColumnSubnetQueries = Map<ColumnSubnetId, number>;
+
 type CachedENR = {
   peerId: PeerId;
   multiaddrTCP: Multiaddr;
   subnets: Record<SubnetType, boolean[]>;
   addedUnixMs: number;
-  custodySubnetCount: number;
+  peerCustodySubnets: number[];
```

Comment:
this ain't the spec right?

Reply:
this is implementation specific and not part of the spec
here we store `peerCustodySubnets` once to reuse later